gh-pages/_docs/06-05-suspicious-metrics.md

```diff
@@ -0,0 +1,14 @@
+---
+
+permalink: /docs/suspicious-metrics/
+title: "Suspicious metrics"
+
+This feature compares the values of different metrics with metric values of 241 Open Source Java projects.
```

Comment:
```suggestion
This feature compares the values of certain metrics from the loaded cc.json file with metric values of several Open Source Java projects.
```

Reply:
I would like to refer to the 241 benchmark projects
"This feature compares the values of certain metrics from the loaded cc.json file with metric values of 241Open Source Java projects."